### PR TITLE
Fixed #619 - Added flag emoji's to language selector component

### LINKED ISSUE #619: Kofta: Make the language codes display in uppercase
grafik

### PR DESCRIPTION
**Pull request by ianjwhite99**
Fixes #619.

- Add's the different flag emoji's to the language selector component. _(JSON provided by @SuperSonicHub1)_

### CODE REVIEW

File: kofta/src/app/components/LanguageSelector.tsx [hunk 1]
```diff
@@ -7,30 +7,26 @@ interface LanguageSelectorProps {
...
-		{ value: "pl", label: "pl" },
-		{ value: "ur", label: "ur" },																															
-		{ value: "pt-BR", label: "pt-br" },
-		{ value: "pt-PT", label: "pt-pt" },
-		{ value: "ru", label: "ru" },
-		{ value: "th", label: "th" },
-		{ value: "tr", label: "tr" },
-		{ value: "zh-CN", label: "zh-cn" },
+		{ "value": "en", "label": "🇬🇧 en" },
+		{ "value": "cs", "label": "🇨🇿 cs" },
```
Comment (YazeedAlKhalaf) [inline kofta/src/app/components/LanguageSelector.tsx]: Arabic is missing in your update,
add this under `en` in the old iteration
`{ value: "ar", label: "🇸🇦 ar" },`

Comment (ianjwhite99) [reply kofta/src/app/components/LanguageSelector.tsx]: Got this added. Thanks for catching it.

**Review by YazeedAlKhalaf — APPROVED** [review]
LGTM 🚀

**Comment by ianjwhite99** [comment]
@YazeedAlKhalaf I noticed there were more languages that were added in some of the recent commits. I went ahead and updated the component to include them.

File: kofta/src/app/components/LanguageSelector.tsx [hunk 2]
```diff
@@ -7,35 +7,35 @@ interface LanguageSelectorProps {
...
+		{ value: "it", label: "🇮🇹 it" },
+		{ value: "ja", label: "🇯🇵 ja" },
+		{ value: "sr", label: "🇸🇷 sr" },
+		{ value: "lt", label: "🇱🇹 lt" },
+		{ value: "nb", label: "🇳🇴 nb" },
+		{ value: "ne", label: "🇳🇵 ne" },
+		{ value: "nl", label: "🇧🇶 nl" },
+		{ value: "pl", label: "🇵🇱 pl" },
+		{ value: "ur", label: "🇵🇰 ur" },
+		{ value: "pt-BR", label: "🇵🇹 pt-br" },
```
Comment (miguelcarreiro) [inline kofta/src/app/components/LanguageSelector.tsx]: `pt-BR` should include the Brazilian flag (🇧🇷), instead of the Portuguese one.

**Comment by ianjwhite99** [comment]
`{ value: "ro", label: "🇷🇴 ro" }` flag has been added, and merge conflicts have been resolved.

**Comment by ermalsh** [comment]
Fix conflicts and add flags to the new languages in LanguageSelector.tsx @ianjwhite99

**Comment by ianjwhite99** [comment]
> Fix conflicts and add flags to the new languages in LanguageSelector.tsx @ianjwhite99

This is taken care of @ermalsh

**Pull request merged**